nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,14 +14,14 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
-[id="run-once-duration-override-operator-release-notes-1-1-1"]
-== Run Once Duration Override Operator 1.1.1
+[id="run-once-duration-override-operator-release-notes-1-2-0"]
+== {run-once-operator} 1.2.0
 
-Issued: 2024-07-01
+Issued: 16 October 2024
 
-The following advisory is available for the {run-once-operator} 1.1.1: link:https://access.redhat.com/errata/RHSA-2024:1616[RHSA-2024:1616]
+The following advisory is available for the {run-once-operator} 1.2.0: (link:https://errata.devel.redhat.com/advisory/135857[RHEA-2024:7548])
```

Comment:
@cbippley could you please help update this to RHSA ?

Reply:
Yes, now that it's been I'll update it right away.